custom-recipes/api-connect/recipe.json

```
@@ -169,16 +169,21 @@
         {
             "name": "pagination_type",
             "label": "Pagination mechanism",
-            "description": "Consult the API's documentation",
+            "description": "Refer to the API's documentation",
             "type": "SELECT",
             "defaultValue": "na",
             "selectChoices":[
                 {"value": "na", "label": "No pagination"},
                 {"value": "next_page", "label": "Next page URL provided"},
                 {"value": "offset", "label": "Offset pagination"},
-                {"value": "page", "label": "Page pagination"}
+                {"value": "page", "label": "Per page"}
             ]
         },
+        {
+            "type": "SEPARATOR",
+            "description": "&#9888;'Per page' mode requires the 'Key to data array' to be defined",
+            "visibilityCondition": "model.pagination_type=='page'"
```

Could you check that "Key to data array" is already filled and if so, remove the warning message ?
```suggestion
            "visibilityCondition": "model.pagination_type=='page' && model.extraction_key"
```

---

👍